userspace/libsinsp/k8s.cpp

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 //
 // k8s.cpp
 //
-#ifndef CYGWING_AGENT
+#ifndef _WIN32
```

Comment:
If you force MINIMAL_BUILD for Windows, is this file even compiled then?

Reply:
I don't think so.

Reply:
fixed. This was put here before the minimal build was created.